src/main/java/uk/gov/hmcts/ccd/domain/service/search/filter/CaseSearchFilter.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.ccd.domain.service.search.filter;
+
+public enum CaseSearchFilter {
```

Comment:
could we remove this class and use instead
CaseDetailsEntity.STATE_FIELD_COL etc...

Reply:
Good point, done